pyproject.toml

```diff
@@ -64,18 +68,22 @@ pytest = ">=7.2"
 coverage = {extras = ["toml"], version = ">7.0"}
 jsonschema = ">=4.10"
 pytest-mock = "^3.11.1"
+ops-scenario = "^6.0.0"
 
 [tool.poetry.group.integration]
 optional = true
 
 [tool.poetry.group.integration.dependencies]
 pytest = ">=7.2"
-juju = "^3.4.0"
+# FIXME (libjuju): Unpin once GH#1093 is fixed
+# (impossible to parse storage constraints)
+juju = "^3.4.0,<3.5.2"
 coverage = {extras = ["toml"], version = ">7.0"}
 pytest-operator = ">0.20"
-pytest-operator-cache = {git = "https://github.com/canonical/data-platform-workflows", tag = "v6.1.1", subdirectory = "python/pytest_plugins/pytest_operator_cache"}
+pytest-operator-cache = {git = "https://github.com/canonical/data-platform-workflows", tag = "v20.0.2", subdirectory = "python/pytest_plugins/pytest_operator_cache"}
 # To be enabled if we are using groups on integration tests
 # pytest-operator-groups = {git = "https://github.com/canonical/data-platform-workflows", tag = "v6.1.1", subdirectory = "python/pytest_plugins/pytest_operator_groups"}
+pytest-microceph = {git = "https://github.com/canonical/data-platform-workflows", tag = "v20.0.2", subdirectory = "python/pytest_plugins/microceph"}
```

Comment:
This is suuuuuper convenient for testing object storage in CI. However, it does not work locally